src/client.js

```diff
@@ -55,11 +55,12 @@ export default class Client {
     // the api-key is on my.myob.com under Adam's account.
     // Need to specify application/json to avoid XML returns.
     const headers = {
-      'x-myobapi-exotoken': this.getUserToken(),
-      'x-myobapi-key': this.clientId,
-      Authorization: `Basic ${this.token.access_token}`,
+      'x-myobapi-exotoken': this.clientId,
+      'x-myobapi-key': process.env.EXO_API_KEY,
```

Comment:
can we change the constructor and pass this in to the constructor (remove token, which isn't used in this SDK).

although environment is best practice for storage of configuration elements it is not great to leak the configuration into the implementation.